api/src/services/users.ts

```diff
@@ -211,6 +216,36 @@ export class UsersService extends ItemsService {
 			if (passwords.length) {
 				await this.checkPasswordPolicy(passwords);
 			}
+
+			if (roles.length) {
+				const increasedCounts: UserCount = {
+					admin: 0,
+					app: 0,
+					api: 0,
+				};
+
+				const existingRoles = [];
+
+				for (const role of roles) {
+					if (typeof role === 'object') {
+						if ('admin_access' in role && role['admin_access'] === true) {
```

Comment:
Since roles here are user provided, do we have to make use of the `toBoolean` util here?
```suggestion
						if ('admin_access' in role && toBoolean(role['admin_access'])) {
```

Reply:
Hmm, I believe our usage of `toBoolean` is for values fetched from DB to account for the inconsistencies. User provided values aren't cast with such magic elsewhere within our codebase. 🤔